docs/rules/static-property-placement.md

```diff
@@ -126,23 +125,23 @@ MyComponent.propTypes = { /*...*/ };
 
 ## Rule Options
 
-```
+```pt
```

Comment:
what is pt?

Reply:
Same as `plaintext`. I could also update these examples to be valid JS and then use js instead.